boa_engine/src/object/jsmap.rs

```diff
@@ -0,0 +1,190 @@
+// This module is a wrapper for the Map Builtin Javascript Object
+//
+// TODO: improve Iterator object interaction: entries, keys, values
+// forEach implementation is missing
```

Comment:
This comment can be remove now right?

Reply:
Removed this comment. Think it's a bit redundant. Although those methods will probably need to be addressed as IteratorRecord is altered.